consensus/service/src/api/client_api_service.rs

```diff
@@ -45,6 +49,9 @@ pub struct ClientApiService {
     is_serving_fn: Arc<(dyn Fn() -> bool + Sync + Send)>,
     authenticator: Arc<dyn Authenticator + Send + Sync>,
     logger: Logger,
+    /// Information kept regarding sessions between clients and consensus
+    /// so that we can drop bad sessions.
+    _tracked_sessions: Arc<Mutex<LruCache<ResponderId, ClientSessionTracking>>>,
```

Comment:
This should be a `ClientSession`. `ResponderId` is used to denote a server's externally-visible `fqdn:port`, so a given consensus node will only have one value (set either via `--listen-uri` or `--client-responder-id`).

Reply:
Ahh, thanks for clarifying!